wsf_scraping/contracts.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+
+from scrapy.contracts import Contract
+from scrapy.exceptions import ContractFail
+
+
+class AjaxContract(Contract):
```

Comment:
What does this allow us to do? Was it not needed in WHO?

Reply:
This is a particular way to use scrapy tests (aka contracts). This one is only needed for Nice because the first request, returning the article list is an Ajax requests. All the other tests (already implemented) only needed basic GET http queries.